app/components/UI/Ramp/Views/GetStarted/GetStarted.test.tsx

```diff
@@ -48,6 +53,11 @@ const mockuseRampSDKInitialValues: Partial<RampSDK> = {
   setIntent: jest.fn(),
 };
 
+const mockUseRegionInitialValues: Partial<ReturnType<typeof useRegions>> = {
+  isDetecting: false,
+  selectedRegion: null,
+};
+
```

Comment:
Second part of this comment: https://github.com/MetaMask/metamask-mobile/pull/14780/files#r2056066559 

```suggestion
const mockUseRegionInitialValues: Partial<ReturnType<typeof useRegions>> = {
  isDetecting: false,
  selectedRegion: null,
};

jest.mock('../../hooks/useRegions', () =>
  jest.fn(() => mockUseRegionInitialValues),
);
```